### PR TITLE
fix: seekbar getting cropped in fullscreen

### PR DESCRIPTION
**Pull request by amar-1995**

### CODE REVIEW

File: packages/roomkit-react/src/Prebuilt/layouts/HLSView.jsx [hunk 1]
```diff
@@ -493,7 +493,7 @@ const HLSView = () => {
                     align="start"
                     css={{
                       position: 'absolute',
-                      bottom: '0',
+                      bottom: isFullScreen ? '2%' : '0',
```
Comment (raviteja83) [inline packages/roomkit-react/src/Prebuilt/layouts/HLSView.jsx:496]: why 2%? to show the seekbar? instead do fullscreen on the video container no. does it work in desktop/mobile?

Comment (amar-1995) [reply packages/roomkit-react/src/Prebuilt/layouts/HLSView.jsx:496]: in rest of the design seekbar is at the top of controls

Comment (amar-1995) [reply packages/roomkit-react/src/Prebuilt/layouts/HLSView.jsx:496]: Yes, 2% is to show seekbar

Comment (raviteja83) [reply packages/roomkit-react/src/Prebuilt/layouts/HLSView.jsx:496]: we can use the height of the seekbar instead no? why 2%? The height won't change right. depending on dvr use thumb also for height

File: packages/roomkit-react/src/Prebuilt/layouts/HLSView.jsx [hunk 2]
```diff
@@ -602,7 +602,7 @@ const HLSView = () => {
       ) : (
         <Flex align="center" justify="center" direction="column" css={{ size: '100%', px: '$10' }}>
           <Flex align="center" gap="2" css={{ position: 'absolute', left: '$4', top: '$4', zIndex: 1 }}>
-            <LeaveRoom screenType={screenType} />
+            {isMobile || isLandscape ? <LeaveRoom screenType={screenType} /> : null}
```
Comment (raviteja83) [inline packages/roomkit-react/src/Prebuilt/layouts/HLSView.jsx:605]: how did we miss this? kind of the first thing in hls view no?

Comment (amar-1995) [reply packages/roomkit-react/src/Prebuilt/layouts/HLSView.jsx:605]: I was checking after starting the stream. So never saw the `stream yet to start` screen.

**Review by raviteja83 — APPROVED** [review]

**Pull request merged**